WIP-SQL/connection-strings.md

```diff
@@ -0,0 +1,111 @@
+# Connection strings
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on connection strings](http://www.dofactory.com/reference/connection-strings)'
+
+---
+## Content
+
+The syntax for conneciton strings is slighty different for all types of servers. There are two main connection types:
```

Comment:
"The syntax for..." syntax means the structure of a language.
use "The structure of connection strings are slightly different for different types of servers"